src/arkode/arkode_butcher_dirk.c

```diff
@@ -600,6 +609,431 @@ ARKodeButcherTable ARKodeButcherTable_LoadDIRK(ARKODE_DIRKTableID imethod)
     B->c[7] = RCONST(1.0);
     break;
 
+  case(ARKODE_ESDIRK324L2SA_4_2_3):   /* ESDIRK3(2)4L[2]SA (A,L stable) */
+#define g   RCONST(0.435866521508458999416019451193556843)
+#define g2  (g*g)
+#define g3  (g2*g)
+#define g4  (g3*g)
+#define g5  (g4*g)
+#define c3  RCONST(0.6)
+    B = ARKodeButcherTable_Alloc(4, SUNTRUE);
+    B->q = 3;
+    B->p = 2;
+
+    B->b[1] = (-RCONST(2.0)+RCONST(3.0)*c3+RCONST(6.0)*g*(RCONST(1.0)-c3))/(RCONST(12.0)*g*(c3-RCONST(2.0)*g));
+    B->b[2] = (RCONST(1.0)-RCONST(6.0)*g+RCONST(6.0)*g2)/(RCONST(3.0)*c3*(c3-RCONST(2.0)*g));
+    B->b[3] = g;
+    B->b[0] = RCONST(1.0) - g - B->b[1] - B->b[2];
+
+    B->d[1] = c3*(-RCONST(1.0)+RCONST(6.0)*g-RCONST(24.0)*g3+RCONST(12.0)*g4-RCONST(6.0)*g5)/(RCONST(4.0)*g*(RCONST(2.0)*g-c3)*(RCONST(1.0)-RCONST(6.0)*g+RCONST(6.0)*g2))
+      + (RCONST(3.0)-RCONST(27.0)*g+RCONST(68.0)*g2-RCONST(55.0)*g3+RCONST(21.0)*g4-RCONST(6.0)*g5)/(RCONST(2.0)*(RCONST(2.0)*g-c3)*(RCONST(1.0)-RCONST(6.0)*g+RCONST(6.0)*g2));
+    B->d[2] = -g*(-RCONST(2.0)+RCONST(21.0)*g-RCONST(68.0)*g2+RCONST(79.0)*g3-RCONST(33.0)*g4+RCONST(12.0)*g5)/(c3*(c3-RCONST(2.0)*g)*(RCONST(1.0)-RCONST(6.0)*g+RCONST(6.0)*g2));
+    B->d[3] = -RCONST(3.0)*g2*(-RCONST(1.0)+RCONST(4.0)*g-RCONST(2.0)*g2+g3)/(RCONST(1.0)-RCONST(6.0)*g+RCONST(6.0)*g2);
+    B->d[0] = RCONST(1.0) - B->d[1] - B->d[2] - B->d[3];
+
+    B->A[1][0] = g;
+    B->A[1][1] = g;
+    B->A[2][1] = c3*(c3-RCONST(2.0)*g)/(RCONST(4.0)*g);
+    B->A[2][0] = c3 - g - B->A[2][1];
+    B->A[2][2] = g;
+    B->A[3][0] = B->b[0];
+    B->A[3][1] = B->b[1];
+    B->A[3][2] = B->b[2];
+    B->A[3][3] = B->b[3];
+
+    B->c[1] = RCONST(2.0)*g;
+    B->c[2] = RCONST(3.0)/RCONST(5.0);
+    B->c[3] = RCONST(1.0);
+    break;
+
+  case(ARKODE_ESDIRK325L2SA_5_2_3):   /* ESDIRK3(2)5L[2]SA (A,L stable) */
+    B = ARKodeButcherTable_Alloc(5, SUNTRUE);
+    B->q = 3;
+    B->p = 2;
+
+    B->c[1] = RCONST(9.0)/RCONST(20.0);
+    B->c[2] = RCONST(9.0)*(RCONST(2.0)+SUNRsqrt(RCONST(2.0)))/RCONST(40.0);
+    B->c[3] = RCONST(3.0)/RCONST(5.0);
+    B->c[4] = RCONST(1.0);
+
+    B->b[0] = (RCONST(2398.0)+RCONST(1205.0)*SUNRsqrt(RCONST(2.0)))/RCONST(2835.0)/(RCONST(4.0)+RCONST(3.0)*SUNRsqrt(RCONST(2.0)));
+    B->b[1] = (RCONST(2398.0)+RCONST(1205.0)*SUNRsqrt(RCONST(2.0)))/RCONST(2835.0)/(RCONST(4.0)+RCONST(3.0)*SUNRsqrt(RCONST(2.0)));
+    B->b[2] = -RCONST(2374.0)*(RCONST(1.0)+RCONST(2.0)*SUNRsqrt(RCONST(2.0)))/RCONST(2835.0)/(RCONST(5.0)+RCONST(3.0)*SUNRsqrt(RCONST(2.0)));
+    B->b[3] = RCONST(5827.0)/RCONST(7560.0);
+    B->b[4] = RCONST(9.0)/RCONST(40.0);
+
+    B->d[0] = (SUNRsqrt(RCONST(2))*RCONST(5547709)-RCONST(4800247))/RCONST(16519545);
```

Comment:
Add `.0` to the integer values in `RCONST` (will fail to build when `realtype` is not `double`)

Reply:
It looks like these were added in when Steven upgraded those coefficients to full quad precision.